src/Data/Bin/Ordering.agda

```diff
@@ -0,0 +1,791 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Ordering on Bin and certain its properties.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Bin.Ordering where
+
+open import Data.Bin.Base
+open import Data.Bin.Properties
+open import Data.Nat as ℕ using (ℕ; z≤n; s≤s)
+open import Data.Nat.Properties as ℕp using (m+n∸m≡n; m+n∸n≡m)
+open import Data.Product using (_,_; proj₁; proj₂)
+open import Data.Sum using (_⊎_; inj₁; inj₂)
+open import Function using (_∘_; _$_; flip; case_of_; _on_)
+open import Level using () renaming (zero to 0ℓ)
+open import Relation.Binary using
```

Comment:
I think with this number of imports we can drop the `using`

Reply:
Done